### PR TITLE
Adding the experimental Video Campaign Performance report to the project

### PR DESCRIPTION
**Pull request by gustavo-moreira**
This is the new feature of importing reports directly from the CSV file to AwReporting. This allows the user to import Video Campaign Reports downloaded from the UI, directly into the DB. The user has just to change the first lines of the CSV file to have one line of text, and then the column headers.

To run the new feature, just add the following arguments:
-onFileReport VIDEO_CAMPAIGN_REPORT -csvReportFile <EDITED_CSV_FILE>

### CODE REVIEW

**Comment by joeltoby** [comment]
Hi Gustavo,

Can you provide a sample CSV for video on the readme please?

Joel

File: aw-reporting-model/src/test/java/com/google/api/ads/adwords/awreporting/model/definitions/ReportVideoCampaignDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  }
+
+  /**
+   * @see com.google.api.ads.adwords.awreporting.model.definitions.
+   *      AbstractReportDefinitionTest#testFirstEntry(com.google.api.ads.adwords.awreporting.model.entities.Report)
+   */
+  @Override
+  protected void testFirstEntry(ReportVideoCampaign first) {
+
+    Assert.assertEquals(7848880157L, first.getAccountId().longValue());
```
Comment (joeltoby) [inline aw-reporting-model/src/test/java/com/google/api/ads/adwords/awreporting/model/definitions/ReportVideoCampaignDefinitionTest.java]: Can we use a fake CID please?

**Comment by joeltoby** [comment]
The 'budget' field from the video CSV is not being persisted correctly to the DB.  It is always 0.00.
Using the sample 'video-campaign-performance.csv' file you provided, here is a sample from my DB:

mysql> select * from AW_ReportVideoCampaign limit 2\G
*************************** 1. row ***************************
               ROW_ID: 3816991963-e-20150315-20150317
           ACCOUNT_ID: 3816991963
             DATE_END: 20150317
      DATE_RANGE_TYPE: CUSTOM_DATE
           DATE_START: 20150315
           PARTNER_ID: NULL
            TIMESTAMP: 2015-05-21 23:21:02
       TOP_ACCOUNT_ID: 5129865483
              ACCOUNT: b
              NETWORK: NULL
     NETWORK_PARTNERS: NULL
          AVERAGE_CPV: 3.00
               BUDGET: 0.00
        CAMPAIGN_NAME: e
                 COST: 4854.00
                  CTR: 0.00
                  DAY: NULL
            DAYOFWEEK: NULL
               DEVICE: NULL
          IMPRESSIONS: 7004
                MONTH: NULL
        MONTH_OF_YEAR: NULL
              QUARTER: NULL
               STATUS: enabled
THUMBNAIL_IMPRESSIONS: 0
    VIDEO_IMPRESSIONS: 7004
      VIEW_PLAYED_100: 27.04
       VIEW_PLAYED_25: 93.45
       VIEW_PLAYED_50: 45.61
       VIEW_PLAYED_75: 33.20
            VIEW_RATE: 26.53
                VIEWS: 1858
       WEBSITE_CLICKS: 30
                 WEEK: NULL
                 YEAR: NULL
*************************** 2. row ***************************
               ROW_ID: 3816991963-f-20150315-20150317
           ACCOUNT_ID: 3816991963
             DATE_END: 20150317
      DATE_RANGE_TYPE: CUSTOM_DATE
           DATE_START: 20150315
           PARTNER_ID: NULL
            TIMESTAMP: 2015-05-21 23:21:02
       TOP_ACCOUNT_ID: 5129865483
              ACCOUNT: b
              NETWORK: NULL
     NETWORK_PARTNERS: NULL
          AVERAGE_CPV: 4.00
               BUDGET: 0.00
        CAMPAIGN_NAME: f
                 COST: 4855.00
                  CTR: 0.00
                  DAY: NULL
            DAYOFWEEK: NULL
               DEVICE: NULL
          IMPRESSIONS: 121767
                MONTH: NULL
        MONTH_OF_YEAR: NULL
              QUARTER: NULL
               STATUS: enabled
THUMBNAIL_IMPRESSIONS: 121767
    VIDEO_IMPRESSIONS: 0
      VIEW_PLAYED_100: 35.10
       VIEW_PLAYED_25: 69.66
       VIEW_PLAYED_50: 50.15
       VIEW_PLAYED_75: 38.02
            VIEW_RATE: 1.02
                VIEWS: 1240
       WEBSITE_CLICKS: 0
                 WEEK: NULL
                 YEAR: NULL
2 rows in set (0.00 sec)

**Comment by gustavo-moreira** [comment]
Fixed the comments.
I will add the instructions on how to run the report after this is merged to master.

**Comment by joeltoby** [comment]
LGTM

**Pull request merged**